src/simulator_core/adapter/transforms/mappers.py

```diff
@@ -44,7 +44,11 @@ def to_entity(self, model: EsdlObject) -> HeatNetwork:
         # create junctions
         # create assets
         # create connection between them
-        assets_list = [EsdlAssetMapper().to_entity(x)
+
+        assets_list = [EsdlAssetMapper().to_entity(x,
+                                                   from_junction=None,
+                                                   to_junction=None, thermal_production=0.0,
+                                                   temperature_supply=0.0)
```

Comment:
Template code?   why not just pass an empty list, or a list with a single object?

Reply:
Removed these arguments, since they where no longer required